rules/functional.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   rules: {
     'functional/immutable-data': 'error',
-    'functional/no-let': ['error', { allowLocalMutation: true }],
-    'functional/no-this-expression': 'error',
+    'functional/no-let': ['error', { allowInFunctions: true }],
```

Comment:
These two got renamed, see [release notes for v5](https://github.com/eslint-functional/eslint-plugin-functional/releases/tag/v5.0.0)